src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs

```diff
@@ -116,11 +116,14 @@ public override ExportResult Export(in Batch<Activity> batch)
                     foreach (var activityLink in activity.Links)
                     {
                         this.WriteLine($"    {activityLink.Context.TraceId} {activityLink.Context.SpanId}");
-                        foreach (var attribute in activityLink.Tags)
```

Comment:
@TimothyMothra FYI you could re-write this as:

```csharp
foreach (ref readonly var attribute in activityLink.EnumerateTagObjects())
```

More efficient and it will handle the null case for you 😄

Might be worth re-writing all of the foreach loops in here to use the newer pattern just to have a good reference doing it correctly. Follow-up thing if you are interested 😉

Reply:
I'll take this as a follow up after this PR merges. :)